### ISSUE TITLE
Support Move command

### ISSUE THREAD

**Issue by tzutalin** [VSCodeVim/Vim#4224]
As a user, I would like to move a line, or a block of lines, with the `:m command` or `:move `

For example, 
I want to use the ':m XX' command to move the current line to the end of the file or the end of the previous line. It can help me refactor my code easily and efficiently.

**Additional context**
https://vim.fandom.com/wiki/Moving_lines_up_or_down#targetText=In%20normal%20mode%20or%20in,to%20move%20the%20block%20up.

**Comment by tzutalin** [comment]
@J-Fields ,
If integrating with neovim, will it support the feature of moving the line?

**Comment by J-Fields** [comment]
@tzutalin Yes. Support without neovim is a reasonable request, though, so I'll keep this issue open.
